conda/activate.py

```diff
@@ -165,8 +173,9 @@ class Help(CondaError):  # NOQA
             from .exceptions import ArgumentError
             raise ArgumentError("invalid command '%s'" % command)
         elif command == 'activate' and len(remainder_args) > 1:
+            print(sys.argv, file=sys.stderr)
```

Comment:
Is this deliberate or a debug statement?

Reply:
yeah there's a problem here in this block